introducing-polyglotweekly.com.md

```diff
@@ -0,0 +1,193 @@
+<!--
+headline: Introducing polyglotweekly.com
+description: introducing the Polyglot Weekly project
+author: Ben Coe
+datePublished: 2015-04-05
+twitter: benjamincoe
+github: bcoe
+-->
+
+It's been almost a year since I wrote an article about programming, this makes me me sad.
+
+> Polyglot Weekly grows from the question, "what would get me blogging more often?"
+
+I love writing. It forces me to step back, reflect on what I've been learning, and helps solidify it.
```

Comment:
I like this, but "and helps solidify it" feels slightly disconnected from the rest of the sentence. Alternatives: 
1. Start a new sentence such as "Writing helps me *solidify* what I have learned."
2. Use an em dash instead of a comma: "...what I've been learning—it helps solidify it."